### PR TITLE
Add Abandoned cart use case

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  Add Abandoned cart use case topic.

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/tutorials/abandoned-cart-analysis.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+---
+id: abandoned-carts
+slug: /abandoned-carts
+title: Abandoned Carts Analysis
+description: Use RisingWave to process and prepare real-time abandon cart data from multiple sources.
+---
+
+## Overview
+
+Abandoned cart data refers to the data collected when customers begin making purchases on an e-commerce platform by adding items to their shopping carts, but do not complete the transaction. This is a valuable resource that businesses can leverage to improve their e-commerce strategies and customer retention rates. By knowing when a customer abandons their purchase, a business can set up real-time notifications to remind the customer to return to their cart and finish the purchase.
```
Comment (hengm3467) [inline docs/tutorials/abandoned-cart-analysis.md]: One possible benefit is businesses can come up with a personalized discount offer for the customer based on the abandoned cart items, to improve the conversion.

File: docs/tutorials/abandoned-cart-analysis.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+To implement real-time capabilities or to perform real-time data transformations, a strong technological infrastructure is necessary. This can be difficult to maintain as there are numerous moving parts and many real-time data transformation tools are tricky to manage and use. However, RisingWave can easily build streaming data pipelines. Data can be ingested from a variety of sources, transformed and aggregated in real-time with SQL queries, and sinked to data warehouses and lakes. To see how easily you can run RisingWave, see [Quick start](/get-started.md).
```
Comment (hengm3467) [inline docs/tutorials/abandoned-cart-analysis.md]: RisingWave can easily build streaming data pipelines doesn't sound correct. RisingWave makes it simple to build real-time data pipelines.

Comment (hengm3467) [reply docs/tutorials/abandoned-cart-analysis.md]: One notable pain point of doing this type of analysis is the need to join streams with batch data (data from databases). This type of join is mentioned in the steps but it's better to call out the difficulty and how RisingWave tackles it with ease.

File: docs/tutorials/abandoned-cart-analysis.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  users u
+ON
+  a.user_id = u.id
+JOIN
+  products p
+ON
+  a.product_id = p.id;
+```
+
+## Summary
```
Comment (hengm3467) [inline docs/tutorials/abandoned-cart-analysis.md:186]: I feel we can add a bit more information here, either workable queries, or more realistic data that can show users that they can retrieve useful data by using materialized views.

**Review by hengm3467 — APPROVED** [review]
LGTM.

**Comment by emile-00** [comment]
Closed as it will be uploaded as a blog

**Pull request closed without merging**